src/nsrt_learning/option_learning.py

```diff
@@ -578,6 +580,33 @@ def _create_regressor(self) -> Regressor:
             grid_num_ticks_per_dim=num_ticks)
 
 
+class _RLOptionLearnerBase(abc.ABC):
```

Comment:
this all looks good. we'll have to think through how we want to reuse the rest of the code in this file in the forthcoming subclasses.